frontend/views/layouts/main.php

```diff
@@ -54,30 +53,31 @@
             . '</li>';
     }
     echo Nav::widget([
-        'options' => ['class' => 'navbar-nav navbar-right'],
+        'options' => ['class' => 'navbar-nav'],
```

Comment:
You may want to add `ml-auto` to push the nav to the right of the header

Reply:
It was deliberate choice.